### PR TITLE
docs: misc updates

### PR DESCRIPTION
**Pull request by jeromelaban**
closes https://github.com/unoplatform/uno/issues/14195

### CODE REVIEW

**Comment by jeromelaban** [comment]
@mergifyio backport release/stable/5.0

File: doc/articles/guides/solution-building-single-targetframework.md [hunk 1]
```diff
@@ -3,9 +3,37 @@ uid: Build.Solution.TargetFramework-override
 ---
 # Improve Build Times with Visual Studio 2022
 
-The Uno Platform template provides a cross-targeted Class library that includes multiple target frameworks. While building with the command line `dotnet build -f net7.0-ios` only builds the application's head and the class library for `net7.0-ios`, Visual Studio builds all the target frameworks, [regardless of the project head's target framework](https://developercommunity.visualstudio.com/t/Building-a-cross-targeted-project-with-m/651372).
+The Uno Platform template provides a cross-targeted Class library that includes multiple target frameworks, and your application may contain your own cross targeted projects as well. This document explain how to make your builds faster.
```
Comment (agneszitte) [inline doc/articles/guides/solution-building-single-targetframework.md:6]:
```suggestion
The Uno Platform template provides a cross-targeted Class library that includes multiple target frameworks, and your application may contain your own cross-targeted projects as well. This document explains how to make your builds faster.
```

File: doc/articles/guides/solution-building-single-targetframework.md [hunk 2]
```diff
@@ -3,9 +3,37 @@ uid: Build.Solution.TargetFramework-override
...
 
-Considering that during development, it is common to work on a single platform at a given time, here's a suggested set of modifications to Uno Platform solution that can be performed on the solution to restrict the active build platform, and therefore significantly speed up build times:
+## Cross-targeted library builds in Visual Studio
+
+While building with the command line `dotnet build -f net7.0-ios` only builds the application's head and the class library for `net7.0-ios`, as of Visual Studio 17.8, class library builds are considering all target frameworks, [regardless of the project head's target framework](https://developercommunity.visualstudio.com/t/Building-a-cross-targeted-project-with-m/651372) being built. _(Please help the Uno Platform community by upvoting the issue!)_
+
+Considering that during development, it is common to work on a single platform at a given time, the two sections below contain suggested set of changes to Uno Platform solution that can be performed on the solution to restrict the active build platform, and therefore significantly speed up build times and make intellisense respond faster.
```
Comment (agneszitte) [inline doc/articles/guides/solution-building-single-targetframework.md:12]:
```suggestion
Considering that during development, it is common to work on a single platform at a given time, the two sections below contain a suggested set of changes to the Uno Platform solution that can be performed on the solution to restrict the active build platform, and therefore significantly speed up build times and make intellisense respond faster.
```

File: doc/articles/guides/solution-building-single-targetframework.md [hunk 3]
```diff
@@ -3,9 +3,37 @@ uid: Build.Solution.TargetFramework-override
...
+
+Choose the section which cover you cases, whether you're using a solution built using Uno Platform 5.0 templates, or if it was created with an earlier version.
```
Comment (agneszitte) [inline doc/articles/guides/solution-building-single-targetframework.md:14]:
```suggestion
Choose the section that covers your cases, whether you're using a solution built using Uno Platform 5.0 templates, or if it was created with an earlier version.
```

File: doc/articles/guides/solution-building-single-targetframework.md [hunk 4]
```diff
@@ -3,9 +3,37 @@ uid: Build.Solution.TargetFramework-override
...
+
+## Improve performance using the Uno Platform templates
+
+When using an Uno Platform 5.0 template, in the **Solution Explorer**, you'll find a folder named **Solution Items**, and a file named `solution-config.props.sample`.
+
+You can follow the directions specified in this file, or follow it here:
```
Comment (agneszitte) [inline doc/articles/guides/solution-building-single-targetframework.md:20]:
```suggestion
You can follow the directions specified in this file, or follow them here:
```

File: doc/articles/guides/solution-building-single-targetframework.md [hunk 5]
```diff
@@ -3,9 +3,37 @@ uid: Build.Solution.TargetFramework-override
...
+- Right-click on the root item (your solution name) and **Open in file explorer**
+- Locate the `solution-config.props.sample` and make a copy named `solution-config.props` in the same folder
+- Back in Visual Studio, and for your convenience you can right-click on `Solution Items` and add `solution-config.props`
+
+Now that we have configured this file, let's say that you want to debug for WebAssembly:
+- In the `solution-config.props` file, uncomment the line which mentions `Wasm` or `WebAssembly` then save it.
+- For the change to take effect, close and reopen the solution, or restart Visual Studio.
```
Comment (agneszitte) [inline doc/articles/guides/solution-building-single-targetframework.md:27]:
```suggestion
Now that we have configured this file, let's say that you want to debug for WebAssembly:
- In the `solution-config.props` file, uncomment the line that mentions `Wasm` or `WebAssembly` then save it.
- For the change to take effect, close and reopen the solution, or restart Visual Studio.
```

File: doc/articles/get-started-vs-2022.md [hunk 6]
```diff
@@ -91,7 +92,7 @@ To create an Uno Platform app:
 1. To debug for **iOS**:
     - Right click on the `MyApp.Mobile` project, select **Set as startup project**
         > [!NOTE]
```
Comment (agneszitte) [inline doc/articles/get-started-vs-2022.md:94]:
```suggestion
1. To debug for **iOS**:
    - Right-click on the `MyApp.Mobile` project, select **Set as startup project**
        > [!NOTE]
```

File: doc/articles/get-started-vs-2022.md [hunk 7]
```diff
@@ -61,14 +61,15 @@ To create an Uno Platform app:
...
-    - **Presentation** gives a choice about using MVVM (e.g. [MVVM Toolkit](https://learn.microsoft.com/en-us/dotnet/communitytoolkit/mvvm/)) or [Uno Platform's MVUX and Feeds](xref:Overview.Reactive.Concept)
-    - **Projects** gives the ability to add a Server project for APIs and hosting for the WebAssembly project
-    - **Testing** provides Unit Testing and [UI Testing projects](https://g7ithub.com/unoplatform/Uno.UITest)
-    - **Features** provides support for WebAssembly PWA and optional [VS Code support](xref:Uno.GetStarted.vscode) files
+    - **Presentation** gives a choice about using MVVM (e.g. [MVVM Toolkit](https://learn.microsoft.com/en-us/dotnet/communitytoolkit/mvvm/)), [Uno Platform's MVUX and Feeds](xref:Overview.Reactive.Concept), or your own.
+    - **Markup** gives the ability to choose between XAML markup or [C# Markup](xref:Extensions.Reference.Markup.GettingStarted).
+    - **Theme** gives the ability to change between [Fluent](xref:uno.themes.fluent.getstarted) and [Material](xref:uno.themes.material.getstarted)
     - **Extensions** allows to choose for [additional Uno.Extensions](xref:Overview.Features) to kickstart your app faster
```
Comment (agneszitte) [inline doc/articles/get-started-vs-2022.md:67]:
```suggestion
    - **Extensions** allows you to choose for [additional Uno.Extensions](xref:Overview.Features) to kickstart your app faster
```

File: doc/articles/get-started-vs-2022.md [hunk 8]
```diff
@@ -61,14 +61,15 @@ To create an Uno Platform app:
 1. You can optionally choose to customize your app based on the sections on the left side:
     - **Framework** allows to choose which `TargetFramework` your app will use. [.NET 7.0](https://dotnet.microsoft.com/en-us/download/dotnet/7.0) is a commonly appropriate choice.
     - **Platforms** provides a list of platforms your application will support. You still can [add additional platforms](xref:Uno.Guides.AddAdditionalPlatforms) later.
```
Comment (agneszitte) [inline doc/articles/get-started-vs-2022.md:63]:
```suggestion
1. You can optionally choose to customize your app based on the sections on the left side:
    - **Framework** allows you to choose which `TargetFramework` your app will use. [.NET 7.0](https://dotnet.microsoft.com/en-us/download/dotnet/7.0) is a commonly appropriate choice.
    - **Platforms** provides a list of platforms your application will support. You still can [add additional platforms](xref:Uno.Guides.AddAdditionalPlatforms) later.
```

File: doc/articles/features/working-with-xaml-hot-reload.md [hunk 9]
```diff
@@ -80,7 +80,7 @@ Hot Reload is supported by Visual Studio for WinAppSDK and provides support in u
 ## Troubleshooting
 
 ### Common issues
-- The application logs file changes. You should see diagnostics messages in the app when a XAML file is reloaded.
+- Observe the application logs, you should see diagnostics messages in the app when a XAML file is reloaded.
 - The file named `RemoteControlGenerator\RemoteControl.g.cs` in the analyzers node for your project contains the connection information, verify that the information host addresses and the port number.
 - WinAppSDK on Windows specific issues
     - Grid Succinct syntax [is not supported](https://github.com/microsoft/microsoft-ui-xaml/issues/7043#issuecomment-1120061686)
```
Comment (agneszitte) [inline doc/articles/features/working-with-xaml-hot-reload.md:86]:
```suggestion
- The file named `RemoteControlGenerator\RemoteControl.g.cs` in the analyzers node for your project contains the connection information, verify that the information host addresses and the port number.
- WinAppSDK on Windows-specific issues
    - Grid Succinct syntax [is not supported](https://github.com/microsoft/microsoft-ui-xaml/issues/7043#issuecomment-1120061686)
```

**Pull request merged**